remote/bitbucketserver/convert.go

```diff
@@ -102,9 +102,12 @@ func convertRepoLite(from *internal.Repo) *model.RepoLite {
 // convertPushHook is a helper function used to convert a Bitbucket push
 // hook to the Drone build struct holding commit information.
 func convertPushHook(hook *internal.PostHook, baseURL string) *model.Build {
+	var branch string
 	//get the ref parts to see if it's a tags or heads
-	refParts := strings.Split(hook.RefChanges[0].RefID, "/")
-	name := refParts[2]
+	refParts := strings.SplitN(hook.RefChanges[0].RefID, "/", 3)
```

Comment:
@tboerger 

The `strings` package doesn't support `StripPrefix` func and we need to consider another prefix condition:

* `refs/heads/feature/branch`
* `refs/tags/v1`

So I add default branch name as empty string and check the `refParts` length to prevent panic attacks.

Thanks.